src/platform/utilities/tests/feature-toggles/helpers.unit.spec.js

```diff
@@ -0,0 +1,68 @@
+import { expect } from 'chai';
+
+const {
+  setCookie,
+  createTokenFromCookie,
+  getFlipperId,
+} = require('../../feature-toggles/helpers');
+
+describe('feature toogles', () => {
+  describe('setCookie', () => {
+    it('should return the cookie if it already exists', () => {
+      const token = 't68mzm0skza1zqwinl4byt';
+      document.cookie = `newcookieName=${token}`;
+      expect(setCookie('newcookieName', 'differentToken')).to.deep.equal(
+        document.cookie.replace(
+          // eslint-disable-next-line no-useless-escape
+          /(?:(?:^|.*;\s*)newcookieName\s*\=\s*([^;]*).*$)|^.*$/,
```

Comment:
If `setCookie` actually changed the cookie, this test wouldn't catch it, I think. I recommend `.to.equal(token);`